src/NServiceBus.Gateway/Routing/Sites/ConfigurationBasedSiteRouter.cs

```diff
@@ -4,6 +4,11 @@ namespace NServiceBus.Gateway.Routing.Sites
 
     class ConfigurationBasedSiteRouter : IRouteMessagesToSites
```

Comment:
There are other files that are using similar namespace paths. Should we fix all of them?